app/crud/country.py

```diff
@@ -9,75 +9,29 @@
-    ) -> Tuple[List[Dict[str, Any]], Dict[str, Any]]:
-        """Retrieve country information and its related research outputs.
+        self,
+        id: str,
+        db: Driver
+    ) ->Dict[str, Any]:
+        """Retrieve country information
 
         Parameters
         ----------
         id : str
             The unique identifier of the country
         db : Driver
             Neo4j database driver instance
-        result_type : Optional[str], default=None
-            Filter outputs by specific result type if provided
+
 
         Returns
         -------
-        Tuple[List[Dict[str, Any]], Dict[str, Any]]
-            A tuple containing:
-            - List[Dict[str, Any]]: List of output dictionaries, each containing:
-                * output details
-                * associated authors
-                * related countries
-            - Dict[str, Any]: Country information dictionary
+        Dict[str, Any]
+            Country information dictionary
         """
-        if result_type:
-            query = """
-                MATCH (o:Output)-[:REFERS_TO]->(c:Country)
-                WHERE c.id = $id AND (o.result_type = $result_type)
-                CALL {
-                    WITH o
-                    MATCH (a:Author)-[r:author_of]->(o)
-                    RETURN a
-                    ORDER BY r.rank
-                }
-                OPTIONAL MATCH (o)-[:REFERS_TO]->(d:Country)
-                RETURN o as outputs,
-                       collect(DISTINCT a) as authors,
-                       collect(DISTINCT d) as countries;
-                """
-            results, summary, keys = db.execute_query(
-                query, id=id, result_type=result_type
-            )
-        else:
-            query = """
-                MATCH (o:Output)-[:REFERS_TO]->(c:Country)
-                WHERE c.id = $id
-                CALL {
-                    WITH o
-                    MATCH (a:Author)-[r:author_of]->(o)
-                    RETURN a
-                    ORDER BY r.rank
-                }
-                OPTIONAL MATCH (o)-[:REFERS_TO]->(d:Country)
-                RETURN o as outputs,
-                       collect(DISTINCT a) as authors,
-                       collect(DISTINCT d) as countries;
-                """
-            results, summary, keys = db.execute_query(query, id=id)
-
-        outputs = [x.data() for x in results]
-        for output in outputs:
-            neo4j_datetime = output["outputs"]["cited_by_count_date"]
-            output["outputs"]["cited_by_count_date"] = datetime.fromtimestamp(
-                neo4j_datetime.to_native().timestamp()
-            )        
-        
         query = """MATCH (c:Country) WHERE c.id = $id RETURN c as country;"""
         results, summary, keys = db.execute_query(query, id=id)
         country = results[0].data()["country"]
-        return outputs, country
+        return country
```

Comment:
Its cleaner to immediately return the value: 
`return results[0].data()["country"]
`